template/Project/version.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this part of the generated template? Why is it here?

Reply:
Hi David, I anticipated this question in my initial message on this pull request. 

The dotnet new templating engine does not allow prerelease templates to be installed. 
Here's the issue: https://github.com/dotnet/templating/issues/1423

Therefore the template needs to be a stable release with no -alpha or similar tags.

As there is already a version.json in the project which stipulates the featherhttp project is in alpha it was necessary to add a new version.json file just for the template project in order to overcome this issue. 

Once FeatherHttp is out of prerelease and you want to have the template version and the FeatherHttp version match then this can be removed/reconsidered

Reply:
https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-preview-1-release-now-available/

```
dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.2.0-preview1.20073.1
```
What am I missing?

Reply:
Nothing at all, I'm incorrect on this. I didn't know you could request the version like that upon installing a new dotnet template - I'll remove the version.json and users will need to stipulate the version as above in order for it to work